python/tvm/relay/frontend/onnx.py

```diff
@@ -4737,6 +4924,9 @@ def _get_convert_map(opset):
         "Elu": Elu.get_converter(opset),
         "Gelu": Gelu.get_converter(opset),
         "BiasGelu": BiasGelu.get_converter(opset),
+        "EmbedLayerNormalization": EmbedLayerNormalization.get_converter(opset),
```

Comment:
If these are all contrib operators, we need a better way to handle different `domains` (see https://github.com/onnx/onnx/blob/main/docs/IR.md#nodes). In this case they are under `com.microsoft` I believe. 

Right now everything is assumed under default namespace.

Just note a TODO here with the namespace they belong under and I will open up an issue.

Reply:
https://github.com/apache/tvm/issues/10950: Issue